DataProcessing/QuiverQuantTwitterFollowersDataDownloader.cs

```diff
@@ -177,25 +165,19 @@ public bool Run()
                                         var universeCsvContents = $"{sid},{ticker},{follower},{dayChange},{weekChange},{monthChange}";
 
                                         _tempData.GetOrAdd(date, new ConcurrentQueue<string>());
-
-                                        if (_tempData.TryGetValue(date, out var tempList))
-                                        {
-                                            tempList.Enqueue(universeCsvContents);
-                                        }
-                                        _tempData.TryUpdate(date, tempList, _tempData[date]);
+                                        _tempData[date].Enqueue(universeCsvContents);
```

Comment:
> _tempData[date]

I think could directly use the returned value of `GetOrAdd` above? avoiding the double access